backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/logging/impl/Log.java

```diff
@@ -733,4 +733,9 @@ SearchException customIndexSettingsJsonSyntaxErrors(String filePath, @Cause Exce
 
 	@Message(id = ID_OFFSET + 139, value = "Unknown filter definition '%1$s'.")
 	SearchException unknownNamedPredicateForSearch(String name, @Param EventContext context);
+
+	@LogMessage(level = Level.WARN)
+	@Message(id = ID_OFFSET + 140, value = "Apply the default limit '%1$s' to the result hits size of '%2$s'." )
```

Comment:
```suggestion
	@Message(id = ID_OFFSET + 140, value = "A search query fetching all hits was requested, but only '%2$s' hits were retrieved because the maximum result window size forces a limit of '%1$s' hits. Refer to Elasticsearch's 'max_result_window_size' setting for more information." )
```